.rubocop.yml

```diff
@@ -8,7 +8,7 @@ AllCops:
   DisplayCopNames: true
 
```

Comment:
![image](https://user-images.githubusercontent.com/93039359/218261320-7bc86f26-0485-4d06-8104-a66f5bfc431b.png)

- [ ] These details are unnecessary :thinking: Kindly remove this part to make your readme file descriptive and professional :+1: